migration/sql-files/029-identities-foreign-key.sql

```diff
@@ -0,0 +1,2 @@
+-- Refactor Identities: add a foreign key constraint
+alter table identities add constraint identies_user_id_users_id_foreign foreign key (user_id) REFERENCES users (id);
```

Comment:
Shouldn't we just set the `user_id` column to `NULL` in the `identities` table when the corresponding user has been deleted ? 
Now, if this is just theorical (because users cannot be removed, but just marked 'deleted'), then I'm fine with both solution. `RESTRICT` seems a safer solution, though.